packages/office-ui-fabric-react/src/components/__snapshots__/TextField.AutoComplete.Example.tsx.shot

```diff
@@ -62,37 +62,6 @@ exports[`Component Examples renders TextField.AutoComplete.Example.tsx correctly
           ms-TextField-wrapper
 
     >
-<<<<<<< HEAD
```

Comment:
I don't see these merge conflict markers committed in master. Not sure what happened here.